### PR TITLE
Grid list and list documentation

### PR DESCRIPTION
**Pull request by Vikew243**
Added documentation about angular objects: list and grid list. The documentation contains a description of objects, html-code and Java-code, as well as links to an overview and java tests.

### CODE REVIEW

File: source/includes/_documentation.md [hunk 1]
```diff
@@ -4854,6 +4854,194 @@ The angular Button:
...
 ---
+## Angular Complex elements 
+
+### List 
+
+#### <a href="https://material.angular.io/components/list/overview" target="_blank">List overview</a>
+
+List is a container component that wraps and formats a series of line items.
+
+There is two different lists in Angular: Basic list and List with sections.
```
Comment (gbuh) [inline source/includes/_documentation.md:4865]: There seems to be a typo: "There is" -> "There are"

File: source/includes/_documentation.md [hunk 2]
```diff
@@ -4854,6 +4854,194 @@ The angular Button:
...
+**text(String text)** | Check whether a text matches a pattern | TextAssert 
+
+#### <a href="https://github.com/jdi-testing/jdi-light/blob/master/jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/complex/ListTests.java" target="_blank">List java tests examples</a>
+### Grid list 
+
+#### <a href="https://material.angular.io/components/grid-list/overview" target="_blank">Grid list overview</a>
+
+Grid list is a two-dimensional list view that arranges cells into grid-based layout. 
+
+There is two different grid lists in Angular: Basic grid list and Dynamic grid list.
```
Comment (gbuh) [inline source/includes/_documentation.md:4957]: There seems to be a typo: "There is" -> "There are"

File: source/includes/_documentation.md (hunk 1 above)
Comment (Vikew243) [reply source/includes/_documentation.md]: Yes. Changes added.

File: source/includes/_documentation.md (hunk 2 above)
Comment (Vikew243) [reply source/includes/_documentation.md]: Yes. Changes added.

**Pull request merged**